bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md

```diff
@@ -5,14 +5,14 @@ Cloud Dataflow.
 
 ## Instructions
 
-Download [the import/export jar](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-dataflow-import/0.9.5.1/bigtable-dataflow-import-0.9.5.1-shaded.jar), which is an aggregation of all required jars.
+Download [the import/export jar](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-dataflow-seqio/0.9.6/bigtable-dataflow-import-0.9.6-shaded.jar), which is an aggregation of all required jars.
 
 ## Export
 
 On the command line:
 
 ```
-java -jar bigtable-dataflow-import-0.9.5.1-shaded.jar export \
+java -jar bigtable-dataflow-import-0.9.6-shaded.jar export \
```

Comment:
The thing I'd like to see changed is the string `bigtable-dataflow-import` in this line and the one below. It looks like this change will not do that?